SpatialGDK/Documentation/content/spatialos-concepts/schema.md

```diff
@@ -7,7 +7,17 @@ Schema is defined in `.schema` files and written in schemalang.  When you use th
 
 #### How to generate schema
 
-To generate schema, select the **Schema** button in the [GDK Toolbar]({{urlRoot}}/content/toolbars#buttons). The GDK automatically iterates through classes with replicated properties to generate the required schema files and then updates the [SchemaDatabase]({{urlRoot}}/content/glossary#schemadatabase).
+**Generating Schema for the first time**
+
+In the Unreal Editor, on the GDK toolbar, open the **Schema** drop-down menu and select **Schema (Full Scan)**. <br/>You must run a full scan the first time you generate Schema for a project. When you select Full Scan, the GDK automatically iterates through all classes with replicated properties to generate the required schema files and then updates the [SchemaDatabase](http://localhost:8080/reference/1.0/content/glossary#schemadatabase). <br/>
```

Comment:
Nits: 
Do we need to say "required schema files" it reads a bit awkwardly?
My pedantry wants to make it "when you select **Schema (Full Scan)**".

```suggestion
In the Unreal Editor, on the GDK toolbar, open the **Schema** drop-down menu and select **Schema (Full Scan)**. <br/>You must run a full scan the first time you generate Schema for a project. When you select **Schema (Full Scan)**, the GDK automatically iterates through all classes with replicated properties to generate the schema files and then updates the [SchemaDatabase](http://localhost:8080/reference/1.0/content/glossary#schemadatabase). <br/>
```